docs/content/en/docs/reference/packagespec/_index.md

```diff
@@ -12,3 +12,4 @@ description: >
 |----------------------------|----------------------------|---------------------------|-----------------------------|
 | [Harbor]({{< relref "./harbor" >}}) | Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. | [v2.5.0]({{< relref "./harbor/v2.5.0.md" >}})<br> [v2.5.1]({{< relref "./harbor/v2.5.1.md" >}}) | https://github.com/goharbor/harbor<br>https://github.com/goharbor/harbor-helm |
 | [MetalLB]({{< relref "./metallb" >}}) | MetalLB is a virtual IP provider for services of type `LoadBalancer` supporting ARP and BGP. | [v0.12.1]({{< relref "./metallb/v0.12.1.md" >}}) | https://metallb.universe.tf/ |
+| [Emissary Ingress]({{< relref "./metallb" >}}) | MetalLB is a virtual IP provider for services of type `Ingress` supporting ARP and BGP. | [v0.12.1]({{< relref "./metallb/v0.12.1.md" >}}) | https://metallb.universe.tf/ |
```

Comment:
Wrong link and this is all metallb stuff.

Reply:
good catch I was copypasting

Reply:
Fixed with a13f307bb95b79204403aab4677946d793a5c9a2